pages/common/adb.md

```diff
@@ -0,0 +1,19 @@
+# Android Debug Bridge
+
+> Android Debug Bridge (adb) is a versatile command line tool
```

Comment:
You already said that it is Android Debug Bridge. No need to repeat it here.
I would rather say about its features.
```md
> Communicate with an emulator instance or connected Android-powered devices
```